recipes-core/packagegroups/packagegroup-ni-runmode.bb

```diff
@@ -20,6 +20,7 @@ RDEPENDS_${PN} = "\
 	glibc-gconv-cp936 \
 	glibc-gconv-iso8859-1 \
 	iproute2-tc \
+	libfmi-dev \
```

Comment:
Similarly, let's break this into its own block and add the same Veristand + Marcelo comment.